osf/models/preprint_provider.py

```diff
@@ -42,13 +42,15 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     allow_submissions = models.BooleanField(default=True)
     additional_providers = fields.ArrayField(models.CharField(max_length=200), default=list, blank=True)
 
-    PREPRINT_WORD_CHOICES = (
```

Comment:
Convention is to have `_CHOICES` variables as 2-tuples.

Rather than breaking this convention, how about adding a separate variable with the plural words:

```python
PREPRINT_WORD_CHOICES = (
    ('preprint', 'Preprint'),
    ('thesis', 'Thesis'),
     # ...
)

PREPRINT_WORD_PLURALS = {
    'preprint': 'preprints',
    'thesis': 'theses',
    # ...
}

@property
def preprint_word_plural(self):
    return self.PREPRINT_WORD_PLURALS[self.preprint_word]
```

Reply:
Well, I made @adlius jump through hoops so that we didn't have to maintain two lists. Can we do PREPRINT_WORDS as the three item tuples and have `preprint_word_singulars` become `preprint_words_choices`? Or am I just making a mess of things and you would much rather have two lists. 

If we do two lists, please add a test to ensure that both of them contain the same items.

Reply:
@brianjgeiger I think your suggestion is fine. @adlius , go ahead with that.